<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-71ONAOR" failures="0" tests="4" name="Default test" time="5.239" errors="0" timestamp="2020-04-27T18:27:35 IST">
  <testcase classname="projects.fox.tests.foxaitest" name="@BeforeClass test" time="0.114">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.baseproject.utility.BasePage.openUrl(BasePage.java:270)
at projects.fox.pages.foxAILoginPage.openLoginPage(foxAILoginPage.java:19)
at projects.fox.tests.foxaitest.test(foxaitest.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass test -->
  <testcase classname="projects.fox.tests.foxaitest" name="test_tour_cashweekly" time="0.0">
    <skipped/>
  </testcase> <!-- test_tour_cashweekly -->
  <testcase classname="projects.fox.tests.foxaitest" name="test_tour_revenue" time="0.0">
    <skipped/>
  </testcase> <!-- test_tour_revenue -->
  <testcase classname="projects.fox.tests.foxaitest" name="test_tour_inventory" time="0.0">
    <skipped/>
  </testcase> <!-- test_tour_inventory -->
  <testcase classname="projects.fox.tests.foxaitest" name="test_target_audience_builder" time="0.0">
    <skipped/>
  </testcase> <!-- test_target_audience_builder -->
</testsuite> <!-- Default test -->
